client/src/screens/CourseSuggestion/CourseSuggestion.js

```diff
@@ -32,18 +32,11 @@ class CourseSuggestion extends Component {
   }
 
   componentDidMount() {
-    this.setState(
-      {
-        studentId: this.props.studentId,
-      },
-      () => {
-        console.log('this is a state', this.state);
-        console.log('this is a prop', this.props.studentId);
-      }
-    );
+    this.setState({
+      studentId: this.props.studentId,

```

Comment:
`this.state.studentID` is not being referenced in this file, maybe we don't need this state. Just use `this.props.studentID`